src/engines/misc/sends/screen.cpp

```diff
@@ -0,0 +1,329 @@
+    ctx.bezierCurveTo(91.6, 149.0, 90.7, 149.9, 90.7, 151.1);
+    ctx.lineTo(90.7, 164.0);
+    ctx.bezierCurveTo(90.7, 165.2, 91.6, 166.1, 92.8, 166.1);
+    ctx.bezierCurveTo(93.9, 166.1, 94.9, 165.2, 94.9, 164.0);
+    ctx.lineTo(94.9, 160.6);
+    ctx.lineTo(98.9, 160.6);
+    ctx.bezierCurveTo(100.0, 160.6, 101.0, 159.7, 101.0, 158.5);
+    ctx.bezierCurveTo(101.0, 157.4, 100.0, 156.5, 98.9, 156.5);
+    ctx.lineTo(94.9, 156.5);
+    ctx.lineTo(94.9, 153.2);
+    ctx.lineTo(103.6, 153.2);
+    ctx.lineTo(106.7, 157.5);
+    ctx.lineTo(103.0, 162.8);
+    ctx.bezierCurveTo(102.3, 163.7, 102.5, 165.1, 103.5, 165.7);
+    ctx.bezierCurveTo(103.8, 166.0, 104.3, 166.1, 104.7, 166.1);
+    ctx.bezierCurveTo(105.3, 166.1, 106.0, 165.8, 106.4, 165.2);
+    ctx.lineTo(109.3, 161.2);
+    ctx.lineTo(112.2, 165.2);
+    ctx.bezierCurveTo(112.6, 165.8, 113.2, 166.1, 113.9, 166.1);
+    ctx.bezierCurveTo(114.3, 166.1, 114.7, 166.0, 115.1, 165.7);
+    ctx.bezierCurveTo(116.0, 165.1, 116.3, 163.7, 115.6, 162.8);
+    ctx.lineTo(111.8, 157.5);
+    ctx.lineTo(115.6, 152.3);
+    ctx.bezierCurveTo(116.3, 151.3, 116.0, 150.0, 115.1, 149.4);
+    ctx.closePath();
+    ctx.fillStyle(Colours::Green);
+    ctx.fill();
+
+    // sends/FX2/FX2dot
+    ctx.beginPath();
+    ctx.moveTo(122.1, 161.2);
+    ctx.bezierCurveTo(120.9, 161.2, 119.9, 162.2, 119.9, 163.4);
+    ctx.bezierCurveTo(119.9, 164.6, 120.9, 165.6, 122.1, 165.6);
+    ctx.bezierCurveTo(123.3, 165.6, 124.3, 164.6, 124.3, 163.4);
+    ctx.bezierCurveTo(124.3, 162.2, 123.3, 161.2, 122.1, 161.2);
+    ctx.closePath();
+    ctx.fill();
+
+    // sends/FX2/FX2dot
+    ctx.beginPath();
+    ctx.moveTo(122.1, 154.0);
+    ctx.bezierCurveTo(123.3, 154.0, 124.3, 153.1, 124.3, 151.9);
+    ctx.bezierCurveTo(124.3, 150.7, 123.3, 149.7, 122.1, 149.7);
+    ctx.bezierCurveTo(120.9, 149.7, 119.9, 150.7, 119.9, 151.9);
+    ctx.bezierCurveTo(119.9, 153.1, 120.9, 154.0, 122.1, 154.0);
+    ctx.closePath();
+    ctx.fill();
+
+    // sends/FX2value
+    ctx.restore();
+    ctx.font(Fonts::Norm, 42);
+    ctx.fillStyle(Colours::White);
+    ctx.fillText(fmt::format("{}", std::round(sendAB_ * 100)), 87.7, 211.3);
+
+    // sends/dryArrow
+
+    // sends/dryArrow/line
+    ctx.save();
+    ctx.beginPath();
+    ctx.moveTo(68.7, 120.0);
+    ctx.lineTo(116.5, 120.0);
+    ctx.bezierCurveTo(140.0, 120.0, 163.1, 125.1, 184.4, 135.0);
+    ctx.lineTo(200.6, 142.5);
+    ctx.lineWidth(6.0);
+    ctx.strokeStyle(Colours::Gray70);
+    ctx.lineCap(LineCap::ROUND);
+    ctx.lineJoin(LineJoin::ROUND);
+    ctx.stroke();
+
+    // sends/dryArrow/arrowhead
+    ctx.beginPath();
+    ctx.moveTo(196.8, 131.7);
+    ctx.lineTo(200.6, 142.6);
+    ctx.lineTo(189.7, 146.3);
+    ctx.strokeStyle(Colours::White);
+    ctx.stroke();
+
+    // sends/dryValue
+    ctx.restore();
+    ctx.font(Fonts::Norm, 42);
+    ctx.fillStyle(Colours::White);
+    ctx.fillText(fmt::format("{}", std::round(mix_ * 100)), 219.3, 211.9);
```

Comment:
prefer doing these calculations in the action handlers, and store the string instead of the `mix_` parameter (or both if needed)

No need to fix it for now, we're replacing this screen anyway

Reply:
You mean the mix_ * 100?

Reply:
no, well that too, but the entire string formatting